src/locale/localed.c

```diff
@@ -84,7 +84,7 @@ static int locale_update_system_manager(Context *c, sd_bus *bus) {
 
         r = sd_bus_call(bus, m, 0, &error, NULL);
         if (r < 0)
-                log_error_errno(r, "Failed to update the manager environment, ignoring: %m");
+                return log_error_errno(r, "Failed to update the manager environment, ignoring: %s", bus_error_message(&error, r));
```

Comment:
let's drop the "ignoring" here. 

In general I think if we say ", ignoring" at the end we should log at most at LOG_WARNING level, and usually return 0 instead of an error.

Reply:
Yeah, but ", ignoring" also tells the user what the outcome was. Just dropping this reduces the usefulness of the message a bit.

Reply:
but we we generate lots of other errors in the same function which we also ignore but don't say so... Dunno what the best solution here is...